docs/installation.md

```diff
@@ -63,54 +107,31 @@ bq mk --table \
   terraform/results_schema.json
 ```
 
-### Cloud Resource Creation - After success
+#### Cloud Resource Creation - After success
 
 You should see a dataset named `pso_data_validator` and a table named
 `results` created inside of your project.
 
-After installing the CLI tool using the instructions below, you will be ready to run data validation commands and output the results to BigQuery. See an example [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/docs/examples.md#store-results-in-a-bigquery-table).
-
-
-## Deploy Data Validation CLI on your machine
-
-The Data Validation tooling requires Python 3.8+.
-
-```
-sudo apt-get install python3
-sudo apt-get install python3-dev
-```
-
-Create and activate a new virtual environment to sandbox the tool and its
-dependencies from your system installation of Python. 
-
-```
-python3.10 -m venv venv
-source venv/bin/activate
-```
-
-Update pip and make sure gcc is installed in your environment.
-```
-sudo apt-get update  && sudo apt-get install gcc -y && sudo apt-get clean
-pip install --upgrade pip
-```
+You are now be ready to run data validation commands and output the results to BigQuery. See an example [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/docs/examples.md#store-results-in-a-bigquery-table).
```

Comment:
```suggestion
You are now ready to run data validation commands and output the results to BigQuery. See an example [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/docs/examples.md#store-results-in-a-bigquery-table).
```

Reply:
Thanks for being diligent here, I obviously didn't proof read the text I was copying.